pack/fhv/fhvc.json

```diff
@@ -90,9 +90,407 @@
+    "type_code": "enemy",
+    "victory": 2
+  },
+  {
+    "code": "10542b",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 15,
+    "enemy_damage": 2,
+    "enemy_evade": 3,
+    "enemy_fight": 4,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": 5,
+    "illustrator": "Helge C. Balzer",
+    "name": "Limulus Hybrid [night]",
+    "pack_code": "fhvc",
+    "position": 42,
+    "quantity": 1,
+    "subname": "In the Dark",
+    "text": "Hunter. Massive. Retaliate.\nLimulus Hybrid gets +3[per_investigator] health.\n<b>Forced</b> - After you successfully attack Limulus Hybrid with a [[Melee]] card: Take 1 damage.",
+    "traits": "Monster. Abomination. Elite",
+    "type_code": "enemy",
+    "victory": 2
+  },
+  {
+    "back_link": "10543b",
+    "code": "10543a",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 16,
+    "enemy_damage": 1,
+    "enemy_evade": 4,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "JB Casacop",
+    "name": "Crustacean Hybrid [day]",
+    "pack_code": "fhvc",
+    "position": 43,
+    "quantity": 2,
+    "subname": "In the Light",
+    "text": "Elusive. Hunter.\nCrustacean Hybrid gets +X health, where X is the current day number.\n<b>Forced</b> - When Crustacean Hybrid would take damage from an attack: Reduce that damage to 1.",
+    "traits": "Creature. Abomination. Elite",
+    "type_code": "enemy",
+    "victory": 0
+  },
+  {
+    "code": "10543b",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 16,
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 4,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "JB Casacop",
+    "name": "Crustacean Hybrid [night]",
+    "pack_code": "fhvc",
+    "position": 43,
+    "quantity": 2,
+    "subname": "In the Dark",
+    "text": "Hunter.\nCrustacean Hybrid gets +X health, where X is the current day number.\n<b>Forced</b> - After Crustacean Hybrid flips to this side: Crustacean Hybrid deals 1 damage to each investigator at its location.",
+    "traits": "Creature. Abomination. Elite",
+    "type_code": "enemy",
+    "victory": 0
+  },
+  {
+    "code": "10544",
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 17,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 2,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "Pixeloid Studios",
+    "name": "Cavern Moss",
+    "pack_code": "fhvc",
+    "position": 44,
+    "quantity": 3,
+    "text": "Aloof. Hunter.\nWhile Cavern Moss is attached to an asset, it may be attacked as if it were an enemy engaged with you./n<b>Forced</b> - After you succeed at a skill test while fighting or investigating at Cavern Moss's location, if it is ready: Attach Cavern Moss to an [[Item]] asset you control. Treat attached asset's text box as if it were blank (except for [[Traits]]).",
```

Comment:
```suggestion
    "text": "Aloof. Hunter.\nWhile Cavern Moss is attached to an asset, it may be attacked as if it were an enemy engaged with you.\n<b>Forced</b> - After you succeed at a skill test while fighting or investigating at Cavern Moss's location, if it is ready: Attach Cavern Moss to an [[Item]] asset you control. Treat attached asset's text box as if it were blank (except for [[Traits]]).",
```